Assets/BossRoom/Scripts/Client/Game/Character/ClientCharacterVisualization.cs

```diff
@@ -242,8 +242,11 @@ void Update()
             }
 
             // NetworkTransform is interpolated - we can just apply it's position value to our visual object
```

Comment:
```suggestion
            // NetworkTransform is interpolated - we can just apply its position value to our visual object
```
While we're here, there was a typo in the above comment